k8s/aat/common/probate/probate-back-office.yaml

```
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-back-office
+  namespace: probate
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:aat-*
```

Unless it is intentional
```suggestion
    fluxcd.io/tag.java: glob:prod-*
```